src/aat/java/uk/gov/hmcts/reform/em/stitching/testutil/Env.java

```diff
@@ -12,7 +12,7 @@
         defaults.setProperty("PROXY", "false");
         defaults.setProperty("TEST_URL", "http://localhost:8080");
         defaults.setProperty("S2S_TOKEN", "AAAAAAAAAAAAAAAA");
-        defaults.setProperty("S2S_SERVICE_NAME", "em_gw");
+        defaults.setProperty("S2S_SERVICE_NAME", "em_stitching_api");
```

Comment:
I think this should be the name of the consumer of the API, so em_ccd_orchestrator